app/component/OfferedRidesPanel.js

```diff
@@ -1,19 +1,91 @@
+import React, { useState, useEffect } from 'react';
 import PropTypes from 'prop-types';
-import React from 'react';
 import OfferedRidesList from './OfferedRidesList';
-import list from '../../static/assets/MOCK_DATA_rides';
+import OfferedRidesDetail from './OfferedRidesDetail';
+import LoginButton from './LoginButton';
+import Loading from './Loading';
+
+let currentRide;
+
+function OfferedRidesPanel(props, context) {
```

Comment:
Arrow all the functions! :)
```suggestion
const OfferedRidesPanel = (props, context) => {
```